lib/ansible/modules/cloud/amazon/ec2_eni.py

```diff
@@ -458,28 +456,32 @@ def detach_eni(eni, module):
         module.exit_json(changed=False, interface=get_eni_info(eni))
 
 
-def find_eni(connection, module):
+def univocally_find_eni(connection, module):
```

Comment:
I suspect 'univocally' should be 'uniquely' or 'unequivocally' (uniquely is likely simpler!)